CMakeLists.txt

```diff
@@ -430,7 +430,9 @@ set(Boost_ADDITIONAL_VERSIONS
   "1.61.1" "1.61.0" "1.61"
   "1.60.1" "1.60.0" "1.60"
   "1.59.1" "1.59.0" "1.59"
-  "1.58.1" "1.58.0" "1.58")
+  "1.58.1" "1.58.0" "1.58"
+  "1.74.0" "1.73"  
+  "17.4.0" "17.4"	)
```

Comment:
Do you mind to add the lines to the top, we are trying to keep the versions in order.